packages/react/src/components/AccountSettings/ConfigureTOTP/defaults.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Alert, Button, Image, TextField } from '../../../primitives';
+import {
+  AccountSettingsError,
+  AccountSettingsImage,
+  AccountSettingsSubmitButton,
+  AccountSettingsTextField,
+} from '../types';
+
+export const ConfirmationCode: AccountSettingsTextField = TextField;
+
+export const SecretKeyQRCode: AccountSettingsImage = Image;
```

Comment:
I like how this makes it easier to understand when just looking at the `ConfigureTOTP` component 👍